packages/google_maps_flutter/google_maps_flutter/ios/Classes/GoogleMapHeatmapController.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Flutter/Flutter.h>
+#import <GoogleMaps/GoogleMaps.h>
+@import GoogleMapsUtils;
+
+// Defines heatmap UI options writable from Flutter.
+@protocol FLTGoogleMapHeatmapOptionsSink
+- (void)setWeightedData:(NSArray<GMUWeightedLatLng *> *)weightedData;
+- (void)setGradient:(GMUGradient *)gradient;
+- (void)setOpacity:(double)opacity;
+- (void)setRadius:(int)radius;
+- (void)setMap;
+@end
+
+// Defines heatmap controllable by Flutter.
+@interface FLTGoogleMapHeatmapController : NSObject <FLTGoogleMapHeatmapOptionsSink>
+@property(atomic, readonly) NSString *heatmapId;
+- (instancetype)initWithHeatmap:(GMUHeatmapTileLayer *)heatmap mapView:(GMSMapView *)mapView;
+- (void)removeHeatmap;
+- (void)clearTileCache;
+@end
+
+@interface FLTHeatmapsController : NSObject
+- (instancetype)init:(FlutterMethodChannel *)methodChannel
+             mapView:(GMSMapView *)mapView
+           registrar:(NSObject<FlutterPluginRegistrar> *)registrar;
+- (void)addHeatmaps:(NSArray *)heatmapsToAdd;
+- (void)changeHeatmaps:(NSArray *)heatmapsToChange;
+- (void)removeHeatmapIds:(NSArray *)heatmapIdsToRemove;
```

Comment:
nits:
rename to: `removeHeatmapsWithIds`

Reply:
Again based on other controllers

Reply:
Yep, I think other controllers need to be updated as well :) But we can at least make this one better.

cc @stuartmorgan for advise, do we prefer making the new code better or making the new code the same style with the old code and fix them all at once.

I will probably just create a PR to fix these styling.

Reply:
Done